src/modules/sub-items/sub.items.module.js

```diff
@@ -315,7 +315,7 @@ export default class SubItemsModule extends Component {
 }
 
 SubItemsModule.propTypes = {
-    parentLocationId: PropTypes.number.isRequired,
+    parentLocationId: PropTypes.string.isRequired,
```

Comment:
LocationId is a number as far as I know.

Reply:
I guest it is number as string. There where errors that `string` is provided instead of `number`. I can check it once more if you want.

Reply:
Can you enforce passing it as a Number? For instance with `parseInt()`

Reply:
In admin-ui we probably passing string instead of Number.

Reply:
Should I change it in `ezplatform-admin-ui`? Or call `parseInt` somewhere in `universal.discovery.module.js`?

Reply:
Should be done in `ezplatform-admin-ui`.

Reply:
You should update the place where you're calling sub items module and there parse the value passed to `parentLocationId` as a Number.

Reply:
Basically, it should be a Number. But fixing it should create a new pull request in admin-ui repo.